projects/11-split.json

```diff
@@ -0,0 +1,29 @@
+{
+    "driver": {
+        "num": 200,
+        "typename": "simpixel"
+    },
+
+    "layout": "strip",
+
+    "aliases": {
+        "bpa": "BiblioPixelAnimations.strip",
+        "Fill": "BiblioPixelAnimations.simple.Fill"
+    },
+
+    "run": {
+        "seconds": 23,
+        "fps": 30
+    },
+
+    "animation": {
+        "typename": "split",
+        "size": 50,
```

Comment:
You have 200 pixels but this 50... is that 50 total? 50 per animation? What if you want a different size per animation.
Also... so this only works with strips? That's fine if so, but how hard would it be to make this work with Matrix? Or crazier yet... a strip and a matrix at the same time?

Reply:
This is already documented in Split.  :-D

            size: a number or a list of numbers representing the size of each
                segment from the original layout.

So you can have arguments like `size=50` or `size=[30, 70, 30, 70]`.  If there aren't enough sizes, the program re-uses sizes from the list, so with 200 LEDs and size 50, you get `[50, 50, 50, 50]`.

Of course, I did think of doing it for non-strips.  But it turns out to be significantly harder because of having to create the "proxy" layout (deciding what width and height are going to be).  

I didn't see a great way to do it so I just did it for strips, and I intend to figure out how to expand it for Matrix in a later change.  (You will note also that the feature request was for strips only.)